vmdb/app/models/ems_kubernetes.rb

```diff
@@ -57,4 +57,17 @@ def authentication_status_ok?(_type = nil)
     # TODO: support real authentication using certificates
     true
   end
+
+  # required by aggregate_hardware
+  def all_computer_system_ids
+    container_nodes.collect(&:computer_system).collect(&:id)
```

Comment:
This needs to preloaded to prevent the N+1 query, with something like...  In addition, the double-collect is overhead you don't need.

```ruby
MiqPreload.preload(container_nodes, :computer_system)
container_nodes.collect { |n| n.computer_system.id }
```